### PR TITLE
Experience level and Time to learn

### PR DESCRIPTION
**Pull request by r-ford**
Removed "Experience level" #101 and changed "Time to learn" as described in #87

### CODE REVIEW

**Review by brian-rose — COMMENTED** [review]
Hi @r-ford, can you also update the template notebook for consistency? i.e. remove the "Experience level" and change the "Time to learn" field.

**Comment by jukent** [comment]
Closes #106

File: appendix/template.ipynb [hunk 1]
```diff
@@ -60,8 +60,7 @@
     "| [Understanding of NetCDF](some-link-to-external-resource) | Helpful | Familiarity with metadata structure |\n",
     "| Project management | Helpful | |\n",
     "\n",
-    "- **Experience level**: with relevant packages or general self-assessed experience as **beginner/intermediate/advanced**\n",
-    "- **Time to learn**: estimate in minutes or qualitatively as **long/medium/short**\n",
+    "- **Time to learn**: estimate in minutes\n",
```
Comment (brian-rose) [inline appendix/template.ipynb:63]: To help future contributors, I suggest including the guidance from #87 here: 
```
- **Time to learn**: estimate in minutes. For a rough idea, use 5 mins per subsection, 10 if longer; add these up for a total. Safer to round up and overestimate."
```

**Review by brian-rose — COMMENTED** [review]
All looks good except my one suggestion on the template.

**Review by brian-rose — APPROVED** [review]
Looks great!

**Comment by brian-rose** [comment]
Just waiting on a review from @ktyle , if you're too busy with classes etc please just let us know and we'll ping another team member.

**Review by ktyle — COMMENTED** [review]
Sorry for the delay in reviewing, but this looks great! One question though: I note the "display_name" now includes "[ipykernel'} in addition to "Python 3". Should that be removed?

**Comment by r-ford** [comment]
> 
> 
> Sorry for the delay in reviewing, but this looks great! One question though: I note the "display_name" now includes "[ipykernel'} in addition to "Python 3". Should that be removed?

@ktyle I think it was added automatically as I made the changes using JupyterLab. I can manually remove it if it might cause some problem.

**Comment by brian-rose** [comment]
> > Sorry for the delay in reviewing, but this looks great! One question though: I note the "display_name" now includes "[ipykernel'} in addition to "Python 3". Should that be removed?
> 
> @ktyle I think it was added automatically as I made the changes using JupyterLab. I can manually remove it if it might cause some problem.

Hmm... it doesn't cause any problems with the test build so I doubt it matters, but let's get some input from @ProjectPythia/infrastructure folks. I'll tag for an extra review.

**Comment by brian-rose** [comment]
Welp, that didn't actually select any individuals. Maybe @dopplershift or @kmpaul could take a quick look.

**Comment by clyne** [comment]
@dcamron says the ipykernel is benign and can be ignored.

**Review by ktyle — APPROVED** [review]
All good and ready to merge, @r-ford !

**Pull request merged**